src/documentation/sdks/python-streaming.md

```diff
@@ -0,0 +1,151 @@
+---
+layout: section
+title: "Apache Beam Python Streaming Pipelines"
+section_menu: section-menu/sdks.html
+permalink: /documentation/sdks/python-streaming/
+---
+
+# Python Streaming Pipelines
+
+Apache Beam SDK for Python supports streaming pipeline execution as of version
```

Comment:
DirectRunner can do this since 2.1.0 and DataflowRunner will start in 2.5.0. Can we try to rephrase as this is experimentally available with some limitations starting from 2.5.0 ?